sim/src/disturbance_experiment.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+##
+#
+# Bash script to measure the disturbance-rejection abilities of the robot.
+# 
+# All this script really does is run `mpc.py` N times and record whether it
+# exits successfully or not. The script failing indicates that the robot fell.
+#
+# Before running this script, verify in config.yaml that:
+#   - The fixed random seed is disabled
+#   - You have selected the correct controller
+#   - Disturbance parameters are set correctly
+#   - The overall simulation duration is set to something reasonably small
+#
+##
+
+num_trials=10
+num_falls=0
+
+# Run the experiment
+for i in $(seq 1 $num_trials); do
+    echo "***Running trial $i/$num_trials ***"
+    python3 mpc.py > /dev/null
+    if [ $? -ne 0 ]; then
+        num_falls=$((num_falls+1))
```

Comment:
I think it would be cool on the assertion of falling line that we save the disturbance that knocked us down. Would be interesting to see which directions we are vulnerable to. This is not a big deal.